engine/hatchery/kubernetes/kubernetes.go

```diff
@@ -0,0 +1,307 @@
+package kubernetes
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/ovh/cds/sdk"
+	"github.com/ovh/cds/sdk/cdsclient"
+	"github.com/ovh/cds/sdk/hatchery"
+	"github.com/ovh/cds/sdk/log"
+
+	"github.com/moby/moby/pkg/namesgenerator"
```

Comment:
don't we have our own namesgenerator ?

Reply:
hum, not since migrate to dep :-/

Reply:
:(

Reply:
don't we have our own namesgenerator package ?